app/controllers/api/messages_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::MessagesController < ApplicationController
+  def random_greeting
+    message = Message.order('RANDOM()').first
+    render json: { greeting: message.content }
```

Comment:
- [ ] Kindly change the property from 'content' which is nil to 'greeting' to render the messages properly.
![image](https://github.com/Divinity-dev/hello-rails-react/assets/91486791/60a12f6a-3074-4836-8855-57987cc22ed8)